app/components/publications/publication/case/info/publication-case-info-panel.js

```diff
@@ -0,0 +1,271 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { isBlank } from '@ember/utils';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { timeout, task, restartableTask } from 'ember-concurrency';
+import moment from 'moment';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class PublicationsPublicationCaseInfoPanelComponent extends Component {
+  @service store;
+  @service intl;
+  @service publicationService;
+
+  @tracked isInEditMode;
+
+  @tracked isViaCouncilOfMinisters;
+
+  @tracked isUrgent;
+
+  @tracked publicationNumberErrorKey;
+  @tracked publicationNumber;
+  @tracked publicationNumberSuffix;
+
+  @tracked numacNumbers;
+  numacNumbersToDelete;
+
+  @tracked decisionDate;
+  @tracked openingDate;
+  @tracked publicationDueDate;
+
+  constructor() {
+    super(...arguments);
+
+    this.initFields();
+  }
+
+  async initFields() {
+    const publicationFlow = this.args.publicationFlow;
+    this.isViaCouncilOfMinisters = await this.getIsViaCouncilOfMinisters(publicationFlow);
+    this.isUrgent = await this.publicationService.getIsUrgent(publicationFlow);
+    this.numacNumbers = publicationFlow.numacNumbers.toArray();
+    this.decisionDate = await this.getDecisionDate(publicationFlow);
+    this.openingDate = publicationFlow.openingDate;
+    this.publicationDueDate = await this.getPublicationDueDate(publicationFlow);
+  }
+
+  async getIsViaCouncilOfMinisters(publicationFlow) {
+    const _case = await publicationFlow.case;
+    const subcases = await _case.subcases;
+    return !!subcases.length;
+  }
+
+  async getDecisionDate(publicationFlow) {
+    const agendaItemTreatment = await publicationFlow.agendaItemTreatment;
+    return agendaItemTreatment.startDate;
+  }
+
+  async getPublicationDueDate(publicationFlow) {
+    const publicationSubcase = await publicationFlow.publicationSubcase;
+    return publicationSubcase.dueDate;
+  }
+
+  @action
+  async putInEditMode() {
+    const publicationFlow = this.args.publicationFlow;
+    this.isInEditMode = true;
+
+    const identification = await publicationFlow.identification;
+    const structuredIdentifier = await identification.structuredIdentifier;
+    this.publicationNumber = structuredIdentifier.localIdentifier;
+    this.publicationNumberSuffix = structuredIdentifier.versionIdentifier;
+
+    this.numacNumbersToDelete = [];
+  }
+
+  @action
+  onChangeIsUrgent(ev) {
```

Comment:
Aangezien dit geen action handler is die als argument meegegeven wordt, de `on` best vermijden.